docs/schema/index-management/index-removal.md

```diff
@@ -7,24 +7,20 @@
 
 ## Overview
 
-Index removal is a two-stage process. In the first stage, one JanusGraph
-signals to all others via the storage backend that the index is slated
-for deletion. This changes the index’s state to `DISABLED`. At that
-point, JanusGraph stops using the index to answer queries and stops
-incrementally updating the index. Index-related data in the storage
-backend remains present but ignored.
-
-The second stage depends on whether the index is mixed or composite. A
-composite index can be deleted via JanusGraph. As with reindexing,
-removal can be done through either MapReduce or JanusGraphManagement.
-However, a mixed index must be manually dropped in the index backend;
-JanusGraph does not provide an automated mechanism to delete an index
-from its index backend.
-
-Index removal deletes everything associated with the index except its
-schema definition and its `DISABLED` state. This schema stub for the
-index remains even after deletion, though its storage footprint is
-negligible and fixed.
+Index removal is a multi-stage process.
+In the first stage, one JanusGraph instance deactivates the index by setting its state to `DISABLED`.
+At that point, JanusGraph stops using the index to answer queries and stops incrementally updating the index.
+Index-related data in the storage backend remains present but ignored.
+
+The second stage depends on whether the index is mixed or composite.
+All composite indices can be deleted via JanusGraph.
+As with reindexing, removal can be done through either MapReduce or JanusGraphManagement.
+However, not all mixed index backends allow automatic removal.
+Index backends for which JanusGraph does not provide an automated deletion mechanism — currently Lucene and Solr — must be manually dropped in the index backend.
+For supported mixed index backends, the removal process is analogue to the composite index removal, with the exception that neither MapReduce nor multi threading in JanusGraphManagement is necessary.
```

Comment:
Please wrap long lines. We usually use 120 as the max length.

Reply:
As I may have stated somewhere else already, I'm a big fan of semantic wrapping (per sentence) because it makes it so much easier to spot the differences in reviews. With a hard cap at 120 characters, changing a single word and re-wrapping makes it look like the entire paragraph has changed.
But if that really is the project's default, I will redo this and wrap the lines.

Reply:
I agree with semantic wrapping, but I think that also wrapping at 120 characters makes sense as longer lines aren't nice to work with in a diff since they are just too long then to be displayed (depending on the screen size & resolution of course).
We usually use 120 characters as a limit in JanusGraph so I think we should stay with that if we don't have good reasons against it. Additionally wrapping each sentence makes sense in my opinion and should already ensure that most lines will be below the 120 char limit.